convex/characterdata/data.ts

```diff
@@ -9,203 +9,203 @@ import { data as f7SpritesheetData } from './spritesheets/f7';
 import { data as f8SpritesheetData } from './spritesheets/f8';
 
 export const Descriptions = [
-    {
```

Comment:
sorry, prettier fixed some other things that got included here. We should get Martin to use Prettier :P